src/inverseRobot.js

```diff
@@ -7,7 +7,17 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const roboNew = {};
```

Comment:
```suggestion
  const reversedRobot = {};
```
Try to provide more descriptive names for your variables